README.md

```diff
@@ -1 +1,109 @@
-# MicroShift
+## MicroShift Upstream Build Process
+
+The RPM/container build process includes the following steps:
+
+1. Replace the MicroShift payload/images with the OKD [released images](https://github.com/okd-project/okd-scos/releases).
+2. Build the MicroShift RPMs and repository from the MicroShift source.
+3. Build the `microshift-okd` Bootc container based on `centos-bootc:stream9`.
+4. Apply upstream customizations (see below).
+
+## Build and Run Microshift upstream without subscription/pull-secret
+
+- Clone the source repositories 
+  ```bash
+  git clone https://github.com/microshift-io/microshift.git microshift-upstream
+  git clone https://github.com/openshift/microshift.git microshift
```

Comment:
Would it be simpler to pull this directory during the container build? We could have an optional build argument called USHIFT_GIT_URL with the default value of `https://github.com/openshift/microshift.git`